google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/run/CloudSdkStartupPolicy.java

```diff
@@ -67,13 +66,11 @@ public String getDisplayString() {
           @Override
           public OSProcessHandler createProcessHandler(
               String workingDirectory, Map<String, String> envVariables) throws ExecutionException {
-            CloudSdkService sdkService = CloudSdkService.getInstance();
 
-            if (sdkService.getSdkHomePath() == null
-                || sdkService.getSdkHomePath().toString().isEmpty()) {
+            CloudSdkService sdkService = CloudSdkService.getInstance();
+            if (sdkService.getSdkHomePath() == null) {
               throw new ExecutionException(
-                  CloudSdkPanel.createErrorMessageWithLink(
-                      GctBundle.message("appengine.cloudsdk.location.missing.message")));
+                  GctBundle.message("appengine.run.server.sdk.misconfigured.message"));
```

Comment:
I don't think we should use the "application server above" message here because this message will actually appear in the event log/balloon notification.
In either of these environments, there should be a browser opening listener, did you try it? If so, we could probably use CloudSdkPanel.createErrorMessageWithLink